src/main/java/uk/gov/hmcts/reform/sendletter/services/date/holidays/response/Event.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.sendletter.services.date.holidays.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDate;
+
+public class Event {
```

Comment:
How about BankHoliday?

Reply:
Do we need this class? I'm wondering if having a set dates wouldn't be enough.

Reply:
just matching the json response names

Reply:
ok